src/kernel/chainparams.cpp

```diff
@@ -349,8 +349,8 @@ class CTestNet4Params : public CChainParams {
         pchMessageStart[3] = 0x28;
         nDefaultPort = 48333;
         nPruneAfterHeight = 1000;
-        m_assumed_blockchain_size = 1;
-        m_assumed_chain_state_size = 0;
+        m_assumed_blockchain_size = 6;
```

Comment:
In 13f2815f4caca68a3ab0d0b865f370956308cad6 "[kernel] update assumed blockchain and chainstate sizes for v29"

I think this is a bit low.

Across my 3 nodes, I have 7.8 GiB, 6.2 GiB, and 9.2 GiB, so I'd expect this to be 11.

Reply:
10% buffer, as stated in the release process doc. `9.2 * 1.1 = 10.12 ~= 11`

Reply:
bumped to 11